quantized_mesh_tile/editable_terrain.py

```diff
@@ -0,0 +1,487 @@
+# -*- coding: utf-8 -*-
+import numpy
+import os
+
+import numpy as np
+from future.utils import old_div
+
+from quantized_mesh_tile import TerrainTile
+from quantized_mesh_tile.llh_ecef import LLH2ECEF
+from quantized_mesh_tile.terrain import MAX, lerp
+from quantized_mesh_tile.utils import triangleArea
+from . import cartesian3d as c3d
+
+null_normal = [0, 0, 0]
+np_null_normal = np.array([0, 0, 0])
+MIN = 0
```

Comment:
put on TerrainTile class

Reply:
and here you define an int, use 0.0 to define a float

Reply:
i thought the quantized values are between 0 and 32767?